vaadin-flow-components-shared-parent/vaadin-flow-components-base/src/main/java/com/vaadin/flow/component/shared/ClientValidationUtil.java

```diff
@@ -23,13 +23,21 @@
  * to valid as a result of client-side validation.
  *
  * @author Vaadin Ltd
+ * @deprecated since 24.6, use
```

Comment:
This would only end up in 24.8, so should we reference that instead?

Reply:
Thanks, fixed.